src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -1001,19 +1054,19 @@ void func_80A76E2C(EnIk* this, GlobalContext* globalCtx, Vec3f* pos) {
         { 900.0, -800.0, 2700.0 },    { 720.0f, 900.0f, 2500.0f },
     };
 
-    if (this->unk_4D4 == 0) {
+    if (this->isAxeSummoned == 0) {
```

Comment:
if it matches,
```suggestion
    if (!this->isAxeSummoned) {
```
and use true/false for it